src/components/MapTiff/MapTiff.tsx

```diff
@@ -205,9 +205,17 @@ const MapTiff = ({
             const rasterMetadata = mapsData.filter(
               (data) => data.fields.id === name,
             )[0];
-            const rasterColors = rasterMetadata.fields.imageData[
-              year
-            ]?.imageParams.map((param: any) => param.color);
+
+            // const rasterColors = rasterMetadata.fields.imageData[
+            //   year
+            // ]?.imageParams.map((param: any) => param.color);
+
+            const rasterColors =
+              name === "cisterna"
+                ? []
+                : rasterMetadata.fields.imageData[year]?.imageParams.map(
+                    (param) => param.color,
+                  );
```

Comment:
Actually liked this solution 'cause it reminds us that this should not be a permanent fix to this "cisterna" problem.

Reply:
Actually, that was the intention lol